jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/TextField.java

```diff
@@ -41,4 +69,8 @@ public TextFieldAssert has() {
         return this.is();
     }
 
+    @JDIAction("Click to focus on '{name}' textfield")
+    public void click() {
+        core().click();
```

Comment:
mb prefer to use smth like this **new Actions(driver).moveToElement(element).click().perform();**

Reply:
I see click() is defined from UI.Element class and our page class is inherited from it. So I think we should reuse this click().

Reply:
ok